examples/experimental/dagster-airlift/dagster_airlift/core/load_defs.py

```diff
@@ -61,75 +45,69 @@ def build_defs_from_airflow_instance(
     else:
         context = DefinitionsLoadContext(load_type=DefinitionsLoadType.INITIALIZATION)
     metadata_key = f"{AIRFLOW_SOURCE_METADATA_KEY_PREFIX}/{airflow_instance.name}"
-    definitions_metadata = _get_or_create_definitions_metadata(
-        context, metadata_key, airflow_instance, defs
-    )
+    serialized_data = _get_or_create_serialized_data(context, metadata_key, airflow_instance, defs)
     return definitions_from_metadata(
-        context, definitions_metadata, defs, airflow_instance, sensor_minimum_interval_seconds
+        serialized_data, defs, airflow_instance, sensor_minimum_interval_seconds
     )
 
 
-def _get_or_create_definitions_metadata(
+def _get_or_create_serialized_data(
     context: DefinitionsLoadContext,
     metadata_key: str,
     airflow_instance: AirflowInstance,
     defs: Definitions,
-) -> "CacheableSpecs":
+) -> "_SerializedData":
     if (
         context.load_type == DefinitionsLoadType.RECONSTRUCTION
         and metadata_key in context.reconstruction_metadata
     ):
-        return deserialize_value(context.reconstruction_metadata[metadata_key], CacheableSpecs)
+        return deserialize_value(context.reconstruction_metadata[metadata_key], _SerializedData)
     else:
-        return compute_cacheable_assets(airflow_instance=airflow_instance, defs=defs)
+        return compute_serialized_data(airflow_instance=airflow_instance, defs=defs)
 
 
-def compute_cacheable_assets(
+def compute_serialized_data(
     airflow_instance: AirflowInstance, defs: Definitions
-) -> "CacheableSpecs":
+) -> "_SerializedData":
```

Comment:
nit: I don't see why we have to prefix these with `_`

Reply:
As a frequent class-auto-complete user, I try to optimize for having as few top-level-exportable things as possible. But no real reason other than that.